### DISCUSSION TITLE
Unable to edit logo

### DISCUSSION THREAD

**Discussion by CloudRider-pixel** [owncast/owncast#1993] [Q&A]
Hi everyone,

I'm completely new with owncast.
I clone the repo, run it locally, and I try to change the logo or  editing the css under the admin page.

Nevertheless, once I upload a new logo, it isn't taken into account and if I refresh the page, mine disappear for the owncast one.
I try under webroot/img/favicon to replace the ms-icon and favicon one with my own. I try also to replace it from data directory without any success.
I'm on Ubuntu 22.04 

In advance, thanks for your help

**Comment by gabek** [comment]
Your browser may have cached your old logo and it hasn't expired. You should try force-refreshing your browser.

As for the CSS, anything you put in the admin for custom CSS will be used. Make sure you save it, however.
